aggregate.go

```diff
@@ -60,6 +76,12 @@ type AggregateMessageHandler interface {
 type AggregateRoot interface {
 	// ApplyEvent updates the aggregate instance to reflect the occurence of an
 	// event that was recorded against this instance.
+	//
+	// It MUST NOT be called with a message of any type that has not been
+	// configured for production by a prior call to Configure().
+	//
+	// It MUST accept all messages ofthe types  that have been configured for
```

Comment:
Missing space `ofthe`, and added space `types  that`:
`// It MUST accept all messages ofthe types  that have been configured for`

Fix:
`// It MUST accept all messages of the types that have been configured for`

Reply:
My spacebar key is packing it in on this new laptop :(

Reply:
You need a space bar button on the touchbar.